recipe-client-addon/test/test_Sampling.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+// Cu is defined in xpc_head.js
+
+Cu.import("resource://gre/modules/Task.jsm", this);
+Cu.import("resource://shield-recipe-client/lib/Sampling.jsm", this);
+
+add_task(function* testStableSample () {
+  // Absolute samples
+  equal(yield Sampling.stableSample("test", 1), true, "stableSample returns true for 100% sample");
```

Comment:
Huh, I didn't realize this was a thing, I thought you needed the `Assert.` prefix. Thanks!